shell/browser/api/electron_api_system_preferences_mac.mm

```diff
@@ -651,13 +654,13 @@ AVMediaType ParseMediaType(const std::string& media_type) {
   return v8::Null(isolate);
 }
 
-void SystemPreferences::SetAppLevelAppearance(gin_helper::Arguments* args) {
+void SystemPreferences::SetAppLevelAppearance(gin::Arguments* args) {
   if (@available(macOS 10.14, *)) {
     NSAppearance* appearance;
     if (args->GetNext(&appearance)) {
       [[NSApplication sharedApplication] setAppearance:appearance];
     } else {
-      args->ThrowError("Invalid app appearance provided as first argument");
+      args->ThrowError();
```

Comment:
We've lost the Better Error here?

Reply:
Yeah, kinda. It matches other gin errors, "Conversion error at index 0". I'm on the fence as to whether consistency or descriptiveness wins here, as the error message here isn't much different to the default one that gin provides.